module_utils/foreman_helper.py

```diff
@@ -209,7 +209,10 @@ def ensure_resource(self, resource, entity_dict, entity, state, check_missing=No
         """ Ensure that a given entity has a certain state """
         changed, changed_entity = False, entity
 
-        if state == 'present':
+        if state == 'present_with_defaults' or state == 'created':
```

Comment:
I guess, this will be up for discussion.
@sean797 @evgeni @Fobhep

Reply:
I would suggest removing `created` for now and keeping these modules consistent. Perhaps we can open another PR and separately changing this across all the modules?

I think I'd prefer not to support both `present_with_defaults` & `created` but I'm happy with whatever if it's supported by others.

Reply:
Ok, I removed it.